### PR TITLE
 KT-21763 Provide Char MIN_VALUE/MAX_VALUE constants

### PR DESCRIPTION
**Pull request by Hiosdra**
https://youtrack.jetbrains.com/issue/KT-21763

### CODE REVIEW

File: libraries/stdlib/src/kotlin/text/Char.kt [hunk 1]
```diff
@@ -37,3 +37,17 @@ public fun Char.equals(other: Char, ignoreCase: Boolean = false): Boolean {
  * Returns `true` if this character is a Unicode surrogate code unit.
  */
 public fun Char.isSurrogate(): Boolean = this in Char.MIN_SURROGATE..Char.MAX_SURROGATE
+
+/**
+ * Minimum value for character
+ * @see <a href="https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html">
```
Comment (ilya-g) [inline libraries/stdlib/src/kotlin/text/Char.kt]: I don't think this way of linking is supported in `@see` tag.  I'm not sure what is the purpose of these links anyway.

Comment (Hiosdra) [reply libraries/stdlib/src/kotlin/text/Char.kt]: Is it good now?

File: core/builtins/native/kotlin/Char.kt [hunk 2]
```diff
@@ -89,18 +89,6 @@ public class Char private constructor() : Comparable<Char> {
...
-         * The minimum value of a character code unit
-         */
-        public val Char.Companion.MIN_VALUE: Char
-            get() = '\u0000'
-
-        /**
-         * The maximum value of a character code unit
-         */
-        public val Char.Companion.MAX_VALUE: Char
-            get() = '\uffff'
```
Comment (ilya-g) [inline core/builtins/native/kotlin/Char.kt]: The extensions are removed, but instead of them there should be const properties in this companion.

**Comment by ilya-g** [comment]
Ok, I've added these constants. Thanks for helping with actual companion object implementations.

See this commit d85c21ba14600a739219547e0813f494b3d691cf in `1.3-M1` branch.

**Pull request closed without merging**